frontend/syntaxmeets/src/components/SyntaxRoom/SyntaxRoom.js

```diff
@@ -1,29 +1,37 @@
-import React, { Fragment, useState, useEffect } from 'react'
-import Navbar from '../Navbar/Navbar';
-import {Grid, Snackbar} from '@material-ui/core';
-import SyntaxEditor from "../SyntaxEditor/SyntaxEditor"
-import SyntaxPad from "../SyntaxPad/SyntaxPad"
+import React, { Fragment, useState, useEffect } from "react";
+import Navbar from "../Navbar/Navbar";
+import { Grid, Snackbar } from "@material-ui/core";
+import SyntaxEditor from "../SyntaxEditor/SyntaxEditor";
+import SyntaxPad from "../SyntaxPad/SyntaxPad";
 import io from "socket.io-client";
 import { Redirect} from "react-router-dom";
 import MuiAlert from '@material-ui/lab/Alert';
 import Footer from '../Footer/Footer';
 
 const socket = io.connect(process.env.REACT_APP_SYNTAXMEETS_BACKEND_API);
 
```

Comment:
Already socket is declared here, why are you defining another socket in line 16, because of that there is a syntax error, and build is failing. 
![image](https://user-images.githubusercontent.com/60866749/110887585-9821cf00-8310-11eb-8d50-ab29b52332a9.png)

Reply:
Apologies , this code had issues after merging . I had not pushed all the updates in the code till then .